Code/CryScriptSystem/ScriptSystem.cpp

```diff
@@ -22,6 +22,8 @@ extern "C"
 #include "ScriptTable.h"
 #include "ScriptUtil.h"
 
+static lua_State *g_L = nullptr;
```

Comment:
Can we use `gEnv->pScriptSystem` instead of adding globals into script system again? Also, this should be in a separate PR.